prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+datasource db {
+  provider = "postgresql"
+  url      = "localhost"
+}
+
+model User {
+  id       String    @unique
+  username String    @unique
+  email    String    @unique
+  password String
+  posts    Post
+  Comment  Comment[]
+}
+
+model Post {
+  id       String  @unique
+  title    String
+  content  String
+  author   User    @relation(fields: [userId], references: [id])
+  comments Comment
+  userId   String
```

Comment:
leaving an example for how schema should look when a field like `question` is a vectorEmbed

```
  question String
  questionAlgorithm String @default("algorithmName")
  questionEmbedding Unsupported("vector[{algorithmName.length}]") // get the algorithmName and length from an enum 
```